.cargo/config

```diff
@@ -0,0 +1,17 @@
+[target.arm-unknown-linux-gnueabihf]
+linker = "arm-linux-gnueabihf-gcc"
+
+[target.armv7-unknown-linux-gnueabihf]
+linker = "arm-linux-gnueabihf-gcc"
+
+[target.aarch64-unknown-linux-gnu]
+linker = "aarch64-linux-gnu-gcc"
+
+[target.arm-unknown-linux-musleabihf]
+linker = "arm-linux-gnueabihf-gcc"
+
+[target.armv7-unknown-linux-musleabihf]
+linker = "arm-linux-gnueabihf-gcc"
+
+[target.aarch64-unknown-linux-musl]
+linker = "aarch64-linux-gnu-gcc"
```

Comment:
```suggestion
[target.'cfg(all(any(target_arch = "arm", target_arch = "armv7"), target_os = "linux"))']
linker = "arm-linux-gnueabihf-gcc"

[target.'cfg(all(target_arch = "aarch64", target_os = "linux"))']
linker = "aarch64-linux-gnu-gcc"

```

After typing this out I'm not sure if it is more or less readable, what do you think?

Reply:
I like it. Original is better for figuring out/testing linkers/dependencies. This is better for readability.

Reply:
Hm... that's weird. I couldn't get cfg to match correctly. 

I guess I'll revert for now. 

`
warning: unused key `linker` in [target] config table `cfg(all(any(target_arch = "arm", target_arch = "armv7"), target_os = "linux"))`
warning: unused key `linker` in [target] config table `cfg(all(target_arch = "aarch64", target_os = "linux"))`
`

https://doc.rust-lang.org/cargo/reference/config.html
sure makes it seem like this should work.

Reply:
Huh, that is strange, I use this feature a lot for embedded rust project, but I could not get it working either.